web/server/codechecker_server/permissions.py

```diff
@@ -715,3 +715,7 @@ def require_manager(permission, extra_params, user):
                                     inherited_from=[PRODUCT_ADMIN,
                                                     PRODUCT_STORE],
                                     managed_by=[PRODUCT_ADMIN])
+
+PRODUCT_VIEW = _create_permission(ProductPermission, 'PRODUCT_VIEW',
+                                  default_enable=False,
+                                  managed_by=[PRODUCT_ADMIN])
```

Comment:
@csordasmarton Do we need to make the `Product_VIEW` inherited from `PRODUCT_ACCESS`